### PR TITLE
Simplified compensation language

### PR DESCRIPTION
**Pull request by hollyallen**
This is to address #254

### CODE REVIEW

File: pages/pay-grades.md [hunk 1]
```diff
@@ -34,29 +34,14 @@ Federal employees are hired at a general schedule (GS) grade from 1 to 15. GS gr
...
+The annual salary cap for all GS employees is $160,300 per year. You cannot be offered more than this under any circumstance. 
 
 Each GS grade level contains a series of 10 steps which also have an impact on compensation. A new GS employee is usually hired at step 1 of the applicable GS grade. However, in special circumstances, agencies may authorize a higher step rate for a newly-appointed federal employee based on a [special need of the agency or superior qualifications of the prospective employee](https://www.opm.gov/policy-data-oversight/pay-leave/pay-administration/fact-sheets/superior-qualifications-and-special-needs-pay-setting-authority/).
 
+**Try the [GS Salary Calculator](https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/2016/general-schedule-gs-salary-calculator/) if you'd like to see how GS leve, step, and locality affects compensation.**
```
Comment (fureigh) [inline pages/pay-grades.md]: @hollyallen, mind if I commit minor copy changes (`leve` --> `level`, `affects` --> `affect`) directly to this PR?

**Comment by fureigh** [comment]
Thanks for this, @hollyallen!

File: pages/pay-grades.md (hunk 1 above)
Comment (hollyallen) [inline pages/pay-grades.md]: please do. I caught the "leve" typo, but not the other.

File: pages/pay-grades.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ The annual salary cap for all GS employees is $160,300 per year. You cannot be o
 
 Each GS grade level contains a series of 10 steps which also have an impact on compensation. A new GS employee is usually hired at step 1 of the applicable GS grade. However, in special circumstances, agencies may authorize a higher step rate for a newly-appointed federal employee based on a [special need of the agency or superior qualifications of the prospective employee](https://www.opm.gov/policy-data-oversight/pay-leave/pay-administration/fact-sheets/superior-qualifications-and-special-needs-pay-setting-authority/).
 
-**Try the [GS Salary Calculator](https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/2016/general-schedule-gs-salary-calculator/) if you'd like to see how GS leve, step, and locality affects compensation.**
+**Try the [GS Salary Calculator](https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/2016/general-schedule-gs-salary-calculator/) if you'd like to see how GS level, step, and locality affects compensation.**
```
Comment (fureigh) [inline pages/pay-grades.md]: Jinx! :) Thanks for fixing.

File: pages/pay-grades.md (hunk 1 above)
Comment (fureigh) [inline pages/pay-grades.md]: :+1:

**Comment by fureigh** [comment]
@rooneywp, want to take a look?

**Comment by pixelwake** [comment]
@fureigh Changes look fine to me.

**Comment by fureigh** [comment]
👍

**Pull request merged**